.yamllint

```diff
@@ -20,3 +20,4 @@ ignore: |
   .github/
   .gitlab/
   .gitlab-ci.yml
+  *etc/kayobe/kolla/config/prometheus/prometheus.yml.d/70-oscapacity.yml
```

Comment:
Why the leading asterisk?

Reply:
I was following convention from other entries in the file. Removing it doesn't seem to break CI.